miio/dreamevacuum_miot.py

```diff
@@ -275,13 +280,11 @@ def reset_sidebrush_life(self) -> None:
         """Reset side brush life."""
         return self.send_action(28, 1)
 
-    def get_properties_for_mapping(self, *, max_properties=15) -> list:
+    def get_properties_for_mapping(self) -> list:
```

Comment:
Revert this (and removal of the todo below) as they are not related to the change.

Reply:
Sure, I will.